mock-hpms/src/main/java/gov/cms/ab2d/hpms/controller/HpmsMockController.java

```diff
@@ -0,0 +1,58 @@
+package gov.cms.ab2d.hpms.controller;
+
+import gov.cms.ab2d.hpms.service.AttestationService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.StreamUtils;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.StringJoiner;
+
+@RestController
+@RequestMapping(path = "/api/cda", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
We decided at some point to not have api in URLs that weren't part of the official API we have for customers, such as HealthAPI and MaintenanceModeAPI, so you can remove the "/api" part.

Reply:
This is a mock service that emulates HPMS.  Their URL includes /api so it is duplicated here.

Reply:
Just make sure it doesn't show up in Swagger page

Reply:
Confirmed.